### DISCUSSION TITLE
how to install locally?

### DISCUSSION THREAD

**Discussion by arunpatro** [coder/code-server#3429] [Q&A]
How can I install this locally to a conda enviroment as I don't have root access?

**Comment by jsjoeio** [comment]
I am not super familiar with Conda so I'm not sure if I can help much there.

Maybe you can your [Conda envrionment with Docker](https://medium.com/@chadlagore/conda-environments-with-docker-82cdc9d25754) and [install code-server via Docker](https://github.com/cdr/code-server/blob/main/docs/install.md#docker)? 

 cc @code-asher @bpmct or @oxy

**Comment by code-asher** [comment]
I think you could install with:

```
conda install -c conda-forge code-server 
```

See https://anaconda.org/conda-forge/code-server

Otherwise using our install script with `--method standalone` would also work. It installs to `~/.local` so it doesn't require root.

**Reply by jsjoeio** [reply]
TIL about `--method standalone`!

**Reply by jsjoeio** [reply]
Also, nice find of the code-server package 🙌

**Reply by jrzoe** [reply]
It's great that this is available on Conda! Are there plans to continue updating the releases provided there? The most recent release is v4.16.1

**Reply by code-asher** [reply]
I am not sure who created it, but we are not maintaining it.  If someone wants to set up a new step in the publish workflow that automatically keeps it up to date, I would be happy to merge it in.
